blog/_posts/2017-02-22-repl-0.6-highlights.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Upgrades to the REPL in v0.6"
+author: <a href="https://kristofferc.github.io/">Kristoffer Carlsson</a>
+---
+
+Since v0.3, Julia comes with a REPL completely written in Julia ([PR #6270](https://github.com/JuliaLang/julia/pull/6270)).
+It contains, among other features, code completion, history with reverse search, a rich set of keybindings which in addition are customizable (see the [Interacting with Julia](http://docs.julialang.org/en/stable/manual/interacting-with-julia/) part of the manual for REPL documentation).
```

Comment:
"It contains, among other features, code completion, history with reverse search, and a rich set of customizable keybindings. (See the manual section Interacting with Julia for REPL documentation.)" reads a bit better imo